Userland/Applications/3DFileViewer/main.cpp

```diff
@@ -571,8 +569,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     window->show();
 
-    auto filename = arguments.argc > 1 ? arguments.argv[1] : "/home/anon/Documents/3D Models/teapot.obj";
-    widget->load_path(filename);
+    auto file = FileSystemAccessClient::Client::the().request_file_read_only_approved(window, filename);
+    if (file.is_error()) {
+        if (file.error().code() != ENOENT)
```

Comment:
If your commit message contains a sentence starting with "Also", that's a good indication that you might need to split the commit up. In this case, I think the `ENOENT` fix should go into a separate commit since it has nothing to do with the `ArgsParser` changes.